README.md

```diff
@@ -1,50 +1,73 @@
-# Submit a New Project to DevNet
-Do you run an open-source project related to Kentico? Submit a Pull Request to add it to the list!
+# Submit a New Kentico EMS Extension to DevNet
 
-Visit the [devnet.kentico.com/open-source](https://devnet.kentico.com/open-source) to see all projects!
+## :warning: Work in progress
+
+Do you run have an open-source project related to Kentico or some extension you want to share? Submit a Pull Request to add it to the list!
+
+Visit the [devnet.kentico.com/marketplace](https://devnet.kentico.com/marketplace) to see all extensions!
 
 ## How to qualify
 
 In order for a project to be registered on DevNet it has to fulfill several requirements, such as:
-- be related to any of the Kentico products
+
+- be related to any of the Kentico EMS products
 - function with the latest version of the related product
```

Comment:
This implies the all extensions will violate this rule the day we release new version, doesn't it?